src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @ngdoc filter
+ * @name pfb.analysis-jobs.status:displayStatus
+ *
+ * @description
+ * Transforms analysis job status into user friendly string
+ */
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function displayStatus() {
+        var statuses = {
+            CREATED: 'Created',
```

Comment:
Yeah, I was wondering how to improve on this. Just copied the pattern from the extant 'batch upload' filters. It is slightly different than a reverse of the map in the service; the descriptive strings here are longer than those used for the drop-downs (defined in the service). Ideally, I think we should be able to get all of these dynamically from the API though.